.travis.yml

```diff
@@ -1,10 +1,15 @@
 language: php
 sudo: false
 
+## Cache composer bits
+cache:
+  directories:
+    - $HOME/.composer/cache
 php:
-  - 5.5
```

Comment:
phpunit only supports `Currently supported` versions of php, so with phpunit 5.x they dropped support for php <= 5.6